common/djangoapps/student/tests/test_email.py

```diff
@@ -233,78 +236,42 @@ def assertFailedRequest(self, response_data, expected_error):
         self.assertEquals(expected_error, response_data['error'])
         self.assertFalse(self.user.email_user.called)
 
-    def test_unauthenticated(self):
-        self.request.user = AnonymousUser()
```

Comment:
The 404 was raised in the wrapper before validate_new_email and do_email_change_request were called-- no longer needed.